doc/src/static/red.css

```diff
@@ -29,6 +29,10 @@
   color: white !important;
 }
 
+p a {
+  text-decoration: underline;
+  text-decoration-style: dotted;
+}
```

Comment:
There are also links under `<li>` etc. But I also know you do not want to change `<a>` globally.